pkg/odo/cli/registry/util/util_test.go

```diff
@@ -69,3 +69,30 @@ func TestIsSecure(t *testing.T) {
 		})
 	}
 }
+
+func TestIsGitBasedRegistry(t *testing.T) {
+	tests := []struct {
+		name        string
+		registryURL string
+		want        bool
+	}{
+		{
+			name:        "Case 1: Returns true if URL contains github",
+			registryURL: "https://github.com/odo-devfiles/registry",
+			want:        true,
+		},
+		{
+			name:        "Case 2: Returns false if URL does not contain github",
+			registryURL: " https://registry.devfile.io",
+			want:        false,
+		},
+	}
```

Comment:
This does not test raw.githubusercontent.com url.

Reply:
I guess we could add a test for that